audit-radiofrance/cockpit.md

```diff
@@ -0,0 +1,233 @@
+# Audit Radio France - Equipe Cockpit
+
+dtc innovation continue son audit chez Radio France.
+Les 11 et 12 septembre, David Bruant est venu pour rencontrer l'équipe "Cockpit" qui s'occupe des différentes bibliothèques front-end communes aux sites des différentes chaînes.
+
+
+## Contexte général
+
+L'équipe Cockpit comporte 4 développeurs et un PO. Elle est née d'une *mitose* d'avec l'équipe Cruiser qui ne garda alors que la responsabilité des données. L'équipe cockpit a commencé avec la responsabilité de maintenir le *player*. De nouveaux projets sont apparus au cours du temps : kami (l'éditeur pour les back-office), damlib (un outil pour inclure des média), un front-end pour l'API, Kaizer (qui contient notamment l'outil pour la grille des programmes) et déjavu (outil qui est un wrapper autour de instaclick + Blazy).
+
+Le temps a mené à une spécialisation où chaque projet est très bien connu d'un développeur (typiquement qui l'a écrit en premier) et très peu des autres.
+
+
+## Objectifs
+
+Les objectifs de l'audit avec l'équipe Cockpit sont les suivants :
+- Evaluer le *[bus factor](https://vimeo.com/187568897)*
+- Evaluer si les différents repos peuvent être distribués en open source
+- Evaluer la facilité pour un nouveau-venu à rentrer dans le code des repos Cockpit
+
+
+## Méthodologie
+
+David Bruant a procédé à des entretiens avec les membres de l'équipe Cockpit (sauf Jérémy, le PO qui était absent ces jours-là)
+
+
+## Avis général
+
+Sans grosse surprise et en continuité de l'avis émis lors du premier morceau d'audit, la première conclusion est que les voyants sont au vert au sein de l'équipe Cockpit. 
+Notamment, l'équipe était déjà consciente des dangers du bus factor et avait déjà mis des actions en places avant l'arrivée de David Bruant pour y remédier.
+
+Aussi, si des efforts supplémentaires seraient à faire du point de vue de la documentation, le fait que les différents repos aient une même *stack* technique (React + redux, modules ES6, etc.) fait qu'il est déjà assez facile pour un débutant pour s'acclimater aux différents projets. Pareillement, la bonne gestion des dépendances permet que les projets pertinents soient quasiment déjà "open-sourçables".
+
+
+## Sujets spécifiques
+
+### Bus factor
+
+Consciente du problème de la spécialisation issu de l'histoire de cette équipe, l'équipe Cockpit avait déjà commencé à mettre en œuvredes pratiques pour limiter le bus factor comme de la transmission d'informations entre deux personnes.
+
+D'autres pratiques pourraient être mises en place et généralisées pour pallier à ce problème :
```

Comment:
[pallier ne prend pas de préposition](https://www.projet-voltaire.fr/regles-orthographe/pallier-quelque-chose-ou-pallier-a-quelque-chose/)

Reply:
s/pallier à/pallier/

Reply:
Wait, the bot is automagically understanding the typo and fixing it?! 😍

Reply:
@davidbgk https://marmelab.com/sedy/ should be a default on GitHub 😉